src/test/java/com/puppycrawl/tools/checkstyle/internal/ImmutabilityTest.java

```diff
@@ -152,7 +152,9 @@ public class ImmutabilityTest {
         "com.puppycrawl.tools.checkstyle.checks.coding.IllegalTokenTextCheck.formatString",
         "com.puppycrawl.tools.checkstyle.checks.javadoc.WriteTagCheck.tagRegExp",
         "com.puppycrawl.tools.checkstyle.checks.naming.AbstractNameCheck.format",
-        "com.puppycrawl.tools.checkstyle.checks.whitespace.AbstractParenPadCheck.option"
+        "com.puppycrawl.tools.checkstyle.checks.whitespace.AbstractParenPadCheck.option",
+        "com.puppycrawl.tools.checkstyle.checks.coding.ConstructorsDeclarationGroupingCheck"
+            + ".allObjBlocks"
```

Comment:
Not ok, this points to fact that you marked Checks steless, but it is stateful by facts.

Reply:
Done. Removed this suppression, explained the reason why this check cannot be stateless here: https://github.com/checkstyle/checkstyle/pull/14749#discussion_r1564656284